packages/blade/src/components/Typography/BaseText/BaseText.stories.tsx

```diff
@@ -1,9 +1,11 @@
+/* eslint-disable react/display-name */
```

Comment:
See this https://github.com/razorpay/blade/pull/451/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R102